BinaryEncoding.md

```diff
@@ -786,6 +950,18 @@ for [future :unicorn:][future multiple tables] use and must be 0 in the MVP.
 | `f32.reinterpret/i32` | `0xbe` | | |
 | `f64.reinterpret/i64` | `0xbf` | | |
 
+## Threads Operations
+
+| Name | Opcode | Immediate | Description |
+| ---- | ---- | ---- | ---- |
+| `i32.wait` | `0xc0` | `offset` | SAB like wait, takes an index + `offset` and an f64 timeout |
+| `i64.wait` | `0xc1` | `offset` | SAB like wait, takes an index + `offset`and an f64 timeout |
+| `i32.wake` | `0xc2` | `offset` | SAB like wake, takes an index + `offset` |
+| `i64.wake` | `0xc3` | `offest` | SAB like wake, takes an index + `offset` |
+| `i32.thread_create` | `0xc4` | `table` | Creates a thread, stores thread in `table` at an i32 index. Returns 0 on success or non-zero otherwise. |
+| `i32.thread_join` | `0xc5` | `table` | Waits to join to thread stored at an i32 index in `table`. Returns 0 on success or non-zero otherwise. |
+| `thread_exit` | `0xc6` | | Halts current thread. |
```

Comment:
It would be good to document this! Is the intent to start off with workers, mimic JS' API, and then add better capabilities later?